requirements/test.txt

```diff
@@ -1,11 +1,13 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --allow-unsafe --output-file=requirements/test.txt --resolver=backtracking --strip-extras requirements/test.in
+#    pip-compile --allow-unsafe --output-file=requirements/test.txt --strip-extras requirements/test.in
 #
```

Comment:
TODO: revert header changes manually

Reply:
I wonder if it's reasonable to add `--no-header`, so this causes less issues in future...

Reply:
That seems reasonable to do in another PR 👍